src/FsCodec.NewtonsoftJson/Codec.fs

```diff
@@ -69,33 +71,36 @@ type Codec private () =
             ///   and an Event Creation <c>timestamp</c>.
             down : 'Context option * 'Event -> 'Contract * 'Meta option * string * string * DateTimeOffset option,
             /// Configuration to be used by the underlying <c>Newtonsoft.Json</c> Serializer when encoding/decoding. Defaults to same as <c>Settings.Create()</c>
-            [<Optional;DefaultParameterValue(null)>]?settings,
+            [<Optional; DefaultParameterValue(null)>] ?settings,
             /// Enables one to fail encoder generation if union contains nullary cases. Defaults to <c>false</c>, i.e. permitting them
-            [<Optional;DefaultParameterValue(null)>]?rejectNullaryCases)
-        : FsCodec.IEventCodec<'Event,byte[],'Context> =
+            [<Optional; DefaultParameterValue(null)>] ?rejectNullaryCases)
+        : FsCodec.IEventCodec<'Event, byte[], 'Context> =
+
         let settings = match settings with Some x -> x | None -> defaultSettings.Value
         let bytesEncoder : TypeShape.UnionContract.IEncoder<_> = new Core.BytesEncoder(settings) :> _
         let dataCodec =
-            TypeShape.UnionContract.UnionContractEncoder.Create<'Contract,byte[]>(
+            TypeShape.UnionContract.UnionContractEncoder.Create<'Contract, byte[]>(
                 bytesEncoder,
-                requireRecordFields=true, // See JsonConverterTests - round-tripping UTF-8 correctly with Json.net is painful so for now we lock up the dragons
-                allowNullaryCases=not (defaultArg rejectNullaryCases false))
-        { new FsCodec.IEventCodec<'Event,byte[],'Context> with
-            member __.Encode(context,event) =
+                requireRecordFields = true, // See JsonConverterTests - round-tripping UTF-8 correctly with Json.net is painful so for now we lock up the dragons
+                allowNullaryCases = not (defaultArg rejectNullaryCases false))
+
+        { new FsCodec.IEventCodec<'Event, byte[], 'Context> with
+            member __.Encode(context, event) =
                 let (c, meta : 'Meta option, correlationId, causationId, timestamp : DateTimeOffset option) = down (context, event)
                 let enc = dataCodec.Encode c
                 let metaUtf8 = meta |> Option.map bytesEncoder.Encode<'Meta>
-                FsCodec.Core.EventData.Create(enc.CaseName, enc.Payload, defaultArg metaUtf8 null, correlationId, causationId, ?timestamp=timestamp)
+                FsCodec.Core.EventData.Create(enc.CaseName, enc.Payload, defaultArg metaUtf8 null, correlationId, causationId, ?timestamp = timestamp)
+
             member __.TryDecode encoded =
                 let cOption = dataCodec.TryDecode { CaseName = encoded.EventType; Payload = encoded.Data }
-                match cOption with None -> None | Some contract -> let union = up (encoded,contract) in Some union }
+                match cOption with None -> None | Some contract -> let union = up (encoded, contract) in Some union }
```

Comment:
maybe inline cOption , and break the match cases into a oneliner for None and 3 lines for the Some?